TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/model/DataProvider.kt

```diff
@@ -9,12 +9,12 @@ package com.microsoft.device.display.samples.twonote.model
 import java.util.ArrayList
 
 object DataProvider {
-    val notes: ArrayList<Note> = ArrayList()
+    val notes: ArrayList<Inode> = ArrayList()
 
     /**
      * Create new note and add it to the top of the note list (position 0)
      */
-    fun createNote() {
-        notes.add(0, Note())
+    fun addNote(note: Inode) {
+        notes.add(note)
```

Comment:
can the first parameter (0) be added back without conflicting with other parts of your implementation (it puts the most recently created note at the top instead of the bottom but we can always change it later if we implement sorting by date modified)

Reply:
the "addInode" method looks at the last index of the list to determine what the next id will be, we could flip it to check the top pretty easily

Reply:
ok cool, maybe we'll just leave that be for now and change it if/when we implement sorting within the notes list